### ISSUE TITLE
Possibility to filter breadcrumbs

### ISSUE THREAD

**Issue by VaclavSir** [ngokevin/redux-raven-middleware#33]
Hi, I would like to know your opinion on a simple feature, that would allow me to filter which actions will be captured to breadcrumbs.

## Usage

It could be a new option:

```
const options = {
  breadcrumbFilter = action => action.type !== 'SOME_BANAL_ACTION',
};

const createStoreWithMiddleware = applyMiddleware(
  RavenMiddleware('my-sentry-dsn', options)
)(createStore);
```

I would be happy to prepare a pull-request if you approve the feature.

## Motivation

I work on an app, that periodically (several times in a second) checks if some external value didn't change from the stored value. Redux way would be to always dispatch an action with current value and move the comparison to the reducer.

But that would generate huge mess in the breadcrumbs, so instead we call some function, that compares current value to the state value and only if it differs, then it dispatches a new action to change the state.

**Comment by captbaritone** [comment]
https://github.com/captbaritone/raven-for-redux/pull/39

**Comment by VaclavSir** [comment]
Thanks, didn't know about this. I think this resolves my issue. 👍

**Comment by captbaritone** [comment]
Just to clarify my rather cryptic comment from earlier (in case any future readers find it confusing) I maintain a different middleware which is similar to Redux Raven Middleware. User @bjudson has contributed a `filterBreadcrumbActions` option to that middleware. The option has been included in version `1.1.0`

Given that Redux Raven Middleware does not seem to be getting any updates, switching to [Raven for Redux](https://github.com/captbaritone/raven-for-redux) might be a good option.
